base-lite/pom.xml

```diff
@@ -0,0 +1,68 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--~
+  ~ Copyright 2017 Confluent Inc.
+  ~
+  ~ Licensed under the Apache License, Version 2.0 (the "License");
+  ~ you may not use this file except in compliance with the License.
+  ~ You may obtain a copy of the License at
+  ~
+  ~ http://www.apache.org/licenses/LICENSE-2.0
+  ~
+  ~ Unless required by applicable law or agreed to in writing, software
+  ~ distributed under the License is distributed on an "AS IS" BASIS,
+  ~ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  ~ See the License for the specific language governing permissions and
+  ~ limitations under the License.
+  ~-->
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+
+    <modelVersion>4.0.0</modelVersion>
+
+    <parent>
+        <groupId>io.confluent</groupId>
+        <artifactId>common-docker</artifactId>
+        <version>7.4.0-0</version>
+    </parent>
+
+    <packaging>pom</packaging>
+
+    <artifactId>cp-base-lite</artifactId>
+
+    <description>Base for new Confluent lightweight Docker images</description>
+    <name>${project.artifactId}</name>
+
+    <properties>
+        <docker.skip-build>false</docker.skip-build>
+        <docker.skip-test>true</docker.skip-test>
```

Comment:
in our meeting we agreed that you would re-write the tests to ensure that your replacement for cub/dub is installed and works. where are those tests and why are they not enabled?

Reply:
@elismaga If this is not a blocker, and we don't have any other comments, can we merge this PR?

Reply:
@elismaga We would like to cover them incrementally in subsequent PRs. Can we get this one merged? That would be of great help for our development as the images will start building via jenkins pipeline, and give us the confidence of end-to-end working.
cc @mohitgrewal

Reply:
In our meeting you told me that you replaced cub/dub with a java utility and that the bash script is only used to run the jar. when i looked the other PR that doesn't seem to be the case.

If you are going to add this new jar and/or bash scripts to multiple images then you should be putting it in the base image. That is why cub/dub were put in the base image. 

Where is the PR for the jar library you mentioned that replaces cub/dub?
This is a blocker and you need to resolve these things before merging.

Reply:
Added the go scripts to replace cub/dub(Earlier I wrote them in bash, but due to difficulty in testing it, used golang)

Reply:
I guess this is resolved as the new go utility is added and reviewed. Please feel free to reopen if this is not the case.